Makefile

```diff
@@ -38,8 +39,10 @@ bin/golangci-lint:
 	GOBIN=$(shell pwd)/bin go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.2
 
 container:
-	docker build -t ${IMAGE_NAME} --build-arg GO_VERSION="${GO_VERSION}" \
```

Comment:
Can we build a `multi-arch` image using buildx?
ref: https://www.docker.com/blog/how-to-rapidly-build-multi-architecture-images-with-buildx/